src/snowflake/snowpark/relational_grouped_dataframe.py

```diff
@@ -121,6 +121,8 @@ def __init__(self, *sets: Union[Column, List[Column]]) -> None:
 class RelationalGroupedDataFrame:
     """Represents an underlying DataFrame with rows that are grouped by common values.
     Can be used to define aggregations on these grouped DataFrames.
+
+    Refer to :meth:`snowflake.snowpark.DataFrame.agg`.
```

Comment:
See also for this?

Reply:
In the whole API doc set, there are 16 instances of using "Refer to " to offer a link. I think it's fine to keep it that way.
```grep -r --color=auto --include \*.html "Refer to " . | wc -l```
Feel free to change it if you want though.

Reply:
Thanks Mary for the searching. I'm good with `See also` here so I changed to it. The other 3 places I prefer "Refer to" because the referred to resources are in the same modules.